src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,17 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `repo   rt-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $(TMPDIR)/%_relevant_signature.txt
```

Comment:
Now only `--onto-name` is needed. Paths to these files can be ascertained. However I've left them as optional CLI args.

Reply:
Ok for now, but generally in cases like this, we always want to see the parameters passed in the python scripts explicitly

Reply:
Ok, that makes sense as an alternative generally and for consistency in the `makefile`s. I added a `todo` about it.